Essentials/src/com/earth2me/essentials/UserData.java

```diff
@@ -698,9 +698,15 @@ public String getLastLoginAddress() {
         return lastLoginAddress;
     }
 
+    // Gives the user an option to store the Player's ip
+    // address in the Player's UUID file
     private void _setLastLoginAddress(String address) {
         lastLoginAddress = address;
-        config.setProperty("ipAddress", address);
+        if (ess.getSettings().storeIPAddressPlainText()) {
+            config.setProperty("ipAddress", address);
+        } else {
+        	config.setProperty("ipAddress", "hidden");
+        }
```

Comment:
I'm not a fan of this method setting the IP as "hidden"

Should set as an empty string or null

Reply:
I had it as null to begin with, although It was suggested to me to have it as hidden, so it also kind of says you can enable it again in the config or something.